### ISSUE TITLE
How to jump with quest 2?

### ISSUE THREAD

**Issue by rogergithubbing** [Gistix/portal2vr#102]
Hi !

Your work is awesome, thank you very much !

By default, I have to press space on the keyboard to jump, or in the xbox compatible controller; I've tried to remap the JUMP action to a quest free button at the portal 2 menu, but nothing happens.

Thanks !

**Comment by Juliasmatius** [comment]
I think its A? Or whatever is the bottom button on the right controller.

**Comment by rogergithubbing** [comment]
> I think its A? Or whatever is the bottom button on the right controller.

Yes, thank you. I'd brought a second hand quest 2 with a defective "a" button, and I have found a sad way to discover that while gaming 😒😊🤞 I've fixed that temporarily by cleaning the contact rubber at the maiboard's controller, but I don't know if it will last... I think I'll buy new rubbers to replace the existing one.
